### PR TITLE
User can now install blocks and kits by referring to respective index

### PR DESCRIPTION
**Pull request by paxperscientiam**
This PR allows end-user to install a block/kit by simply referring to its respective index number. 

Index numbers are derived from alphabetic listing of keys in JSON manifests for blocks/kits.

Example implementation:
Given such a listing as this:
````bash
|18:09:58|ramos@Prometheus:[test]> node ../bin/foundation.js kits list
 1) blog: 13 blocks
 2) dashboard: 16 blocks
 3) ecommerce: 19 blocks
 4) marketing: 15 blocks
 5) mobile-app: 15 blocks
 6) news: 22 blocks
 7) portfolio: 11 blocks
 8) real-estate-travel: 13 blocks
````

One could install the "marketing" kit in the following way:
````bash
|18:10:13|ramos@Prometheus:[test]> node ../bin/foundation.js kits install 4
You don't appear to be in a ZURB stack project, so we can't automatically install building blocks
? Do you want to download as a zip file? Yes
downloaded marketing.zip
````
I will updated README in a moment.

### CODE REVIEW

**Comment by paxperscientiam** [comment]
poke @kball 
poke @IamManchanda

**Comment by IamManchanda** [comment]
:+1:

**Comment by paxperscientiam** [comment]
@harry were you able to test? Worked well for you?

File: lib/commands/blocks.js [hunk 1]
```diff
@@ -37,37 +37,53 @@ function listBuildingBlocks(args, options, callback) {
...
-          });
-        }
+  fetchUrl('foundation.zurb.com', "/building-blocks/data/building-blocks.json", function(data) {
+    var name = args[0];
+    if (Number.isInteger(1*name)) {
+      var selection = 1*name;
+      var i = 0;
+      var kits = JSON.parse(data);
+      _.each(kits, function(value, key) {
+	i++;
```
Comment (kball) [inline lib/commands/blocks.js]: Indentation?

File: lib/commands/blocks.js [hunk 2]
```diff
@@ -37,37 +37,53 @@ function listBuildingBlocks(args, options, callback) {
...
+	if (selection === i) {
+	  name = key;
+	}
       });
     }
-  });
+
+    assertInstallableRepo(function(type) {
+      if(type === 'zip') {
+	downloadZipFile(name, function(err) {
```
Comment (kball) [inline lib/commands/blocks.js]: indentation

**Comment by kball** [comment]
Lets clean up the indentation on this and then happy to merge it.

File: lib/commands/blocks.js (hunk 2 above)
Comment (paxperscientiam) [inline lib/commands/blocks.js]: soon!

File: lib/commands/blocks.js (hunk 1 above)
Comment (paxperscientiam) [inline lib/commands/blocks.js]: soon!

**Comment by paxperscientiam** [comment]
@kball sorry to be dense, but would you mind clarifying what the problem with the indentation is?

**Pull request closed without merging**